source/fundamentals.txt

```diff
@@ -25,20 +25,20 @@ Fundamentals
    /fundamentals/csfle
    /fundamentals/time-series
 
-The Fundamentals section contains detailed information to help you
-expand your knowledge on how to use MongoDB with the Java driver. We
-divide the information in this section into the following topics: 
+The Fundamentals section is where you will find coverage of
+core operations. This section contains information on how to:
```

Comment:
A couple items about this paragraph:

1) I the think "core" adjective in this context is used as an opinion. We should avoid opinion/editorial in our writing and try to keep it objective.
2) This list is introduced using a fragment rather thana sentence which we should avoid per the style guide: https://docs.mongodb.com/meta/style-guide/style/lists/#introductory-text